jdbcSource/build.gradle

```diff
@@ -48,6 +48,12 @@ tasks.withType(Test) {
     if (rootProject.hasProperty("EntConJDBCURL")) {
         systemProperty "EntConJDBCURL", rootProject.findProperty("EntConJDBCURL") ?: "empty"
     }
+    if (rootProject.hasProperty("EntConTestSourceName")) {
+        systemProperty "EntConTestSourceName", rootProject.findProperty("EntConTestSourceName") ?: "empty"
```

Comment:
What's the purpose of the "empty"?  Seems like we've already checked for existence.  Moreover, in TestJDBCBase.java, lines 31 & 32 (ish), we only check for null.  So if this sets it to empty, then we'll create sources & types named "empty" which isn't, I don't think, the intent.  Seems like there's no deed for a default value here since we've already verified presence -- it won't change in the meantime in this case.